src/taco-cli/test/buildAndRunTelemetry.ts

```diff
@@ -291,7 +291,12 @@ module BuildAndRunTelemetryTests {
             (typeof telemetryProperties === "undefined").should.be.equal(false);
             validateGzipedSize(telemetryProperties, "ios", iosExpectedGzipedSize);
             validateGzipedSize(telemetryProperties, "android", androidGzipSize);
-            telemetryProperties.should.eql(expected); // We are comparing the objects, after overriding the sizes with the expected values
```

Comment:
yes because test is written in a way where it executes for build/run/emulate. Run I believe has subcommand in it, rest two don't have it.